src/css/components/_ripple.scss

```diff
@@ -1,12 +1,12 @@
 @keyframes ripple-pop {
-  from {
+  0% {
```

Comment:
@jakearchibald We can disable this stylelint rule if you prefer the old way (it's the same to me)